highfidelity/hifi
# Cellscience Updates

This PR changes cellscience to use an AC script to move its entities around, instead of an entity script. 

To test:

1. Visit hifi://cellscience
2. Observe that the cells rotate
3. Click "Inside the Cell" and observe that the vesicles (greyish sphere things) move.
